protocols/virtualfund/virtualfund.go

```diff
@@ -580,7 +580,8 @@ func (o *Objective) proposeLedgerUpdate(connection Connection, sk *[]byte) (prot
 		return protocols.SideEffects{}, err
 	}
 
-	message := protocols.CreateSignedProposalMessage(connection.Channel)
+	recipient := ledger.Follower()
```

Comment:
On L573 we refer to "proposer" but I think it should be "leader".

Reply:
Resolved in https://github.com/statechannels/go-nitro/pull/638/commits/76932e82ce4448c025e2b45fa04b543497dc4aa5